apps/rush-lib/src/start.ts

```diff
@@ -1,6 +1,25 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+import * as colors from 'colors';
+import * as semver from 'semver';
+
+// check to ensure that we are using a supported version of NodeJS, otherwise write a warning
+if (semver.satisfies(process.versions.node, '< 6.0.0')) {
+  console.error(colors.red(`You are using an outdated version of Node ("${process.versions.node}").`
+    + ` You should upgrade to Node 6 or Node 8.`));
```

Comment:
Your version of Node.js (1.2.3) is very old and not supported by Rush.  Please upgrade to the latest Long Term Support (LTS) version.